local-server/local-server-services/src/main/resources/fxml/LocalServerServices.fxml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import javafx.collections.FXCollections?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.control.TextArea?>
+<?import javafx.scene.layout.BorderPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+
+<BorderPane fx:controller="com.esri.samples.localserver.LocalServerServicesController"
+           xmlns:fx="http://javafx.com/fxml">
+  <!-- Top Section -->
+  <top>
+    <HBox spacing="10">
+      <padding><Insets top="10" right="10" bottom="5" left="10"/></padding>
+    
+      <VBox spacing="3"> 
+        <Label text="Local Server Controls"/>
+        <Button fx:id="btnStartServer" text="Start Local Server" minWidth="200" onAction="#handleStartStopLocalServer"/>
+        <Button fx:id="btnStopServer" text="Stop Local Server" minWidth="200" onAction="#handleStartStopLocalServer" 
+          disable="true"/>
+      </VBox>
+    
+      <VBox HBox.hgrow="ALWAYS" spacing="3"> 
+        <Label text="Local Services Controls"/>
+        <ComboBox fx:id="serviceOptions" maxWidth="Infinity">
+          
+          <value>
+            <String fx:value="Map Service"/>
+          </value>
+          
+          <items>
+            <FXCollections fx:factory="observableArrayList">
+              <String fx:value="Map Service"/>
+              <String fx:value="Feature Service"/>
+              <String fx:value="Geoprocessing Service"/>
+            </FXCollections>
+          </items>
+        </ComboBox>
+        
+        <HBox spacing="3">
+          <Button fx:id="btnStartService" HBox.hgrow="ALWAYS" text="Start Service" maxWidth="Infinity" 
+            onAction="#handleStartService" disable="true"/>
+          <Button fx:id="btnStopService"  HBox.hgrow="ALWAYS" text="Stop Service" maxWidth="Infinity" 
+            onAction="#handleStopService" disable="true"/>
+        </HBox>
+ 
+      </VBox>
+    </HBox>
+  </top>
+    
+    <!-- Center Section -->

```

Comment:
The xml tag gives the same information. Maybe if these were more descriptive, like: "Service status log" and "running services"